javascript-arrays/main.js

```diff
@@ -0,0 +1,23 @@
+const colors = ['red', 'white', 'blue'];
+
+for (const i in colors) {
+  console.log('value of colors[' + i + ']:', colors[i]);
+}
```

Comment:
`for..in` loops are meant for objects. They work, because arrays are technically objects, but they aren't the correct way to loop over objects. You would want to use a normal `for` loop instead, or a `for..of` loop.